### PR TITLE
Fix dead docs links

### PR DESCRIPTION
**Pull request by tas50**
These have been broken for a while

### CODE REVIEW

File: docs/user-manual.md [hunk 1]
```diff
@@ -827,7 +834,7 @@ Note: This is not possible immediately after performing the operator upgrade.
    kubectl get mondooauditconfigs.v1alpha1.k8s.mondoo.com mondoo-client -n mondoo-operator -o yaml > audit-config.yaml
    ```
 
-2. Map the old `v1alpha1` config to the new `v1alpha2` and save the new `MondooAuditConfig`. Find the mapping from `v1alpha1` to `v1alpha2` [here](../api/v1alpha1/mondooauditconfig_types.go#L155-L199).
```
Comment (czunker) [inline docs/user-manual.md:830]: I don't think the last sentence makes sense any more when we remove the `here`.

Comment (tas50) [reply docs/user-manual.md:830]: @czunker Is there a place we can continue to link folks?

Comment (czunker) [reply docs/user-manual.md:830]: I just checked the history. We removed the v1alpha1 API 2 years ago:
image

I think we can remove the complete
```
### I had a `MondooAuditConfig` in my cluster with version `v1alpha1` and now I can no longer access it. What should I do?
```

Comment (tas50) [reply docs/user-manual.md:830]: I nuked it all. Thanks for looking around

**Review by czunker — APPROVED** [review]
Thanks @tas50

**Pull request merged**